client/my-sites/current-site/notice.jsx

```diff
@@ -236,22 +236,20 @@ export class SiteNotice extends React.Component {
 		return (
 			<UpsellNudge
 				event="calypso_upgrade_nudge_impression"
-				forceDisplay={ true }
+				forceDisplay
 				tracksClickName="calypso_upgrade_nudge_cta_click"
 				tracksClickProperties={ eventProperties }
 				tracksImpressionName="calypso_upgrade_nudge_impression"
 				tracksImpressionProperties={ eventProperties }
-				callToAction={ ctaText || 'Upgrade' }
+				callToAction={ ctaText || translate( 'Upgrade' ) }
 				href={ `/plans/${ site.slug }?discount=${ name }` }
 				title={ bannerText }
 			/>
 		);
 	}
 
 	promotionEndsToday( { endsAt } ) {
-		const now = new Date();
-		const format = 'YYYYMMDD';
-		return moment( now ).format( format ) === moment( endsAt ).format( format );
+		return moment().isSame( endsAt, 'day' );
```

Comment:
Since `endsAt` is a simple `Date` object, we could easily avoid using `moment` by comparing just the date portion of the two date objects. We could do something like:

```suggestion
		return endsAt.toDateString() === new Date().toDateString();
```

Reply:
As this way of comparing dates is not terribly elegant, I have another idea: what if we used this opportunity to introduce our first usage of `date-fns`? That would look like:
```js
import { isSameDay } from 'date-fns';

const same = isSameDay( endsAt, new Date() );
```

Otherwise, I'd rather keep the `moment` usage so that it can be easily found and replaced with `date-fns` in the future.

Reply:
I agree it's not that elegant, but it's native JS, it's clear and it works 😉  The options according to my personal preference are:

* `Date.prototype.toDateString()` - because it removes a usage of `moment` without introducing another library
* `moment` - because it leaves the code as it was from the perspective of library usage
* `date-fns` - because it introduces a new library (even if it's considered a better option than `moment`)

I'll leave it to your judgment as to what to go with, since I don't have a strong opinion on it and don't want to block the PR.

Reply:
I think that having to use `.toDateString()` shows gaps in the native APIs that will hopefully be addressed at some time. I'll stay with `moment` here so that future upgrade can be easily grepped. Thanks for your feedback :+1: